packages/components/tabs/__tests__/tabs.test.tsx

```diff
@@ -0,0 +1,670 @@
+// @ts-nocheck
+import { nextTick, ref } from 'vue'
+import { mount } from '@vue/test-utils'
+import { describe, expect, test, vi } from 'vitest'
+import { EVENT_CODE } from '@element-plus/constants'
+import Tabs from '../src/tabs'
+import TabPane from '../src/tab-pane.vue'
+import TabNav from '../src/tab-nav'
+
+describe('Tabs.vue', () => {
+  test('create', async () => {
+    const wrapper = mount(() => (
+      <Tabs>
+        <TabPane label="label-1">A</TabPane>
+        <TabPane label="label-2">B</TabPane>
+        <TabPane label="label-3" ref="pane-click">
+          C
+        </TabPane>
+        <TabPane label="label-4">D</TabPane>
+      </Tabs>
+    ))
+
+    const tabsWrapper = wrapper.findComponent(Tabs)
+    const navWrapper = wrapper.findComponent(TabNav)
+    const panesWrapper = wrapper.findAllComponents(TabPane)
+    await nextTick()
+
+    const navItemsWrapper = navWrapper.findAll('.el-tabs__item')
+
+    expect(navItemsWrapper[0].classes('is-active')).toBe(true)
+    expect(panesWrapper[0].classes('el-tab-pane')).toBe(true)
+    expect(panesWrapper[0].attributes('id')).toBe('pane-0')
+    expect(panesWrapper[0].attributes('aria-hidden')).toEqual('false')
+    expect(tabsWrapper.vm.$.exposed.currentName.value).toEqual('0')
+
+    await navItemsWrapper[2].trigger('click')
+    expect(navItemsWrapper[0].classes('is-active')).toBe(false)
+    expect(panesWrapper[0].attributes('aria-hidden')).toEqual('true')
+    expect(navItemsWrapper[2].classes('is-active')).toBe(true)
+    expect(panesWrapper[2].attributes('aria-hidden')).toEqual('false')
+    expect(tabsWrapper.vm.$.exposed.currentName.value).toEqual('2')
+  })
+
+  test('active-name', async () => {
+    const activeName = ref('b')
+    const handleClick = (tab) => {
+      activeName.value = tab.paneName
+    }
+    const wrapper = mount(() => (
+      <Tabs model-value={activeName.value} onTabClick={handleClick}>
```

Comment:
## Suggest

It's very trivial, but for consistency I think it would be nice to use `v-model`!